src/Utils/Notification.js

```diff
@@ -0,0 +1,43 @@
+import React, { Component } from 'react';
+import Snackbar from 'material-ui/Snackbar';
+import IconError from "material-ui/svg-icons/alert/error-outline";
+import IconSuccess from "material-ui/svg-icons/navigation/check";
+import {css} from 'aphrodite';
+import AppStyles from "./../AppStyles";
+import config from "./../config/config"
+
+class ConfirmationMessage extends Component {
```

Comment:
Why is this class not named like the file it is in? This could lead to confusions.